src/contracts/adapters/configuration-adapter.js

```diff
@@ -109,3 +109,31 @@ export const parseConfigs = (inputs) => {
   if (configs.debug) console.log(configurations);
   return configurations;
 };
+
+const getKey = (key) => {
+  const kycOnboardingKeys = new Set([
+    "kyc-onboarding.signerAddress",
+    "kyc-onboarding.chunkSize",
+    "kyc-onboarding.unitsPerChunk",
+    "kyc-onboarding.maximumChunks",
+    "kyc-onboarding.maximumTotalUnits",
+    "kyc-onboarding.maxMembers",
+    "kyc-onboarding.canTopUp",
+    "kyc-onboarding.fundTargetAddress",
+    "kyc-onboarding.tokensToMint",
```

Comment:
Ideally we would read all the config names from tribute-contracts confg-utils.ts or something like that, but for now it is ok to define them here.